code/items/ICarriableItem.cs

```diff
@@ -5,7 +5,8 @@ public enum SlotType
         Primary = 1,
         Secondary,
         Melee,
-        Equipment,
+        OffensiveEquipment,
```

Comment:
What is an offensive equipment item? Aren'g offensive items weapons?

Reply:
Offensive being the slot for outward attacking items, C4, Knife, Newton Launcher. Utility for other stuff such as decoy, noisemaker and health station to give a few examples.